### PR TITLE
Add release notes to upgrade modal

### PR DESCRIPTION
**Pull request by tzarebczan**
Adding back in release notes based on the old upgrade modal: https://github.com/lbryio/lbry-app/blob/master/src/renderer/modal/modalUpgrade/view.jsx#L25

I did this based on inspection, not sure how it will look. Main difference here is that it's all in a section.

### CODE REVIEW

**Comment by tzarebczan** [comment]
Igor gave this a shot and it looked wonky on first try:
screen_shot_2018-03-30_at_19 47 41

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "LBRY",
-  "version": "0.21.2",
+  "version": "0.20.0",
```
Comment (seanyesmunt) [inline package.json]: Guessing you didn't mean to add this

File: src/renderer/modal/modalAutoUpdateDownloaded/view.jsx [hunk 2]
```diff
@@ -32,6 +30,13 @@ class ModalAutoUpdateDownloaded extends React.PureComponent {
               'A new version of LBRY has been released, downloaded, and is ready for you to use pending a restart.'
             )}
           </p>
+          <p className="meta text-center">
+            {__('Want to know what has changed?')} See the{' '}
+            <Button
```
Comment (seanyesmunt) [inline src/renderer/modal/modalAutoUpdateDownloaded/view.jsx]: add `button="link"`

File: package.json (hunk 1 above)
Comment (IGassmann) [reply package.json]: This was done by me in order to test the modal. Has to be undone, naturally.

**Comment by seanyesmunt** [comment]
Fixed here https://github.com/lbryio/lbry-app/pull/1288
screen shot 2018-04-16 at 6 51 02 pm

**Pull request closed without merging**